2-copy-of-code/lesson-13/scripts/amazon.js

```diff
@@ -98,8 +111,19 @@ document.querySelectorAll('.js-add-to-cart')
 
       addedMessage.classList.add('added-to-cart-visible');
 
-      setTimeout(() => {
```

Comment:
Why do we have one setTimeout inside another setTimeout in here?

Reply:
it is just one 
the one in red was deleted and replaced with the code in green